addons/vehiclelock/functions/fnc_handleVehicleInitPost.sqf

```diff
@@ -27,7 +27,7 @@ TRACE_1("params",_vehicle);
 
     params ["_vehicle"];
 
-    if ((_vehicle isKindOf "Car") || {_vehicle isKindOf "Tank"} || {_vehicle isKindOf "Helicopter"}) then {
+    if (_vehicle isKindOf "Car" || {_vehicle isKindOf "Tank" || {_vehicle isKindOf "Helicopter" || {_vehicle isKindOf "Plane"}}}) then {
```

Comment:
Can't we combine them into IsKindOf Air?

Reply:
```suggestion
    if (-1 < ["Car", "Tank", "Helicopter", "Plane"] findIf {_vehicle isKindOf _x}) then {
```
github suggestion is cool :)

Reply:
Like, except the -1 should be at the end and the < be a >=.

Reply:
this whole line is pointless
func is only called from xeh added to specific vehicle types
https://github.com/acemod/ACE3/blob/master/addons/vehiclelock/CfgEventHandlers.hpp#L23